friprosveta/views.py

```diff
@@ -478,6 +479,46 @@ def allocations_json(request, timetable_slug=None):
     filtered_allocations = _allocation_set(param_ids,
                                            tt.allocations,
                                            request.user.is_staff)
+
+    #returns more information about allocations
+    r = request.GET
+    if 'mode' in r:
+        l = r['mode']
+        if l == 'ext':
+            AllocationVM = namedtuple('AllocationVM', ['object', 'day_index', 'hour_index', 'duration'])
```

Comment:
What does AllocationVM stand for?

Reply:
Renamed variables to better explain their meaning.